bikespace_dashboard/js/components/map.js

```diff
@@ -1,140 +1,141 @@
     attribution: [
       '&copy; Maps ',
       '<a href="https://www.thunderforest.com/">',
-        'Thunderforest',
+      'Thunderforest',
       '</a>, ',
       '&copy; Data ',
       '<a href="https://www.openstreetmap.org/copyright">',
-        'OpenStreetMap contributors',
+      'OpenStreetMap contributors',
       '</a>',
-    ].join(""),
+    ].join(''),
   },
   openstreetmap: {
-    url: "https://tile.openstreetmap.org/{z}/{x}/{y}.png",
+    url: 'https://tile.openstreetmap.org/{z}/{x}/{y}.png',
     attribution: [
       '&copy; ',
       '<a href="https://www.openstreetmap.org/copyright">',
-        'OpenStreetMap contributors',
+      'OpenStreetMap contributors',
       '</a>',
-    ].join(""),
+    ].join(''),
   },
 };
 
 class Map extends Component {
   constructor(parent, root_id, shared_state) {
     super(parent, root_id, shared_state);
-    
+
     // initialize map and zoom to City of Toronto
     this.lmap = L.map('map').setView([43.733399, -79.376221], 11);
     L.tileLayer(tiles.thunderforest_atlas.url, {
-      attribution: tiles.thunderforest_atlas.attribution
+      attribution: tiles.thunderforest_atlas.attribution,
     }).addTo(this.lmap);
 
     this.markers = this.buildMarkers();
     this.lmap.addLayer(this.markers);
 
     // improve keyboard navigation
-    $(document).on("keydown", ".marker-cluster", function(e) {
-      if (e.key == "Enter" || e.key == " ") {
+    $(document).on('keydown', '.marker-cluster', e => {
+      if (e.key === 'Enter' || e.key === ' ') {
         $(document.activeElement).trigger('click');
       }
     });
-    $(document).on("keydown", ".leaflet-marker-icon", function(e) {
-      if (e.key == " ") { // Enter already works
+    $(document).on('keydown', '.leaflet-marker-icon', e => {
+      if (e.key === ' ') {
+        // Enter already works
         $(document.activeElement).trigger('click');
       }
     });
 
     // analytics
-    this.lmap.on('popupopen', (e) => {
+    this.lmap.on('popupopen', e => {
       super.analytics_event(`${this.root_id}_${e.type}`, {
         submission_id: e.popup.submission_id,
       });
     });
   }
 
   buildMarkers() {
-    let markers = L.markerClusterGroup();
+    const markers = L.markerClusterGroup();
 
     // BUILD POPUP CONTENT
     // pre-generate template for each issue type
     const duration_descr = {
-      'minutes': "for <strong>less than an hour</strong>",
-      'hours': "for <strong>several hours</strong>",
-      'overnight': "<strong>overnight</strong>",
-      'multiday': "for <strong>several days</strong>"
+      minutes: 'for <strong>less than an hour</strong>',
+      hours: 'for <strong>several hours</strong>',
+      overnight: '<strong>overnight</strong>',
+      multiday: 'for <strong>several days</strong>',
     };
-    let issue_tags = {};
-    for (let entry of Object.values(ia)) {
+    const issue_tags = {};
```

Comment:
Same as earlier, `issue_tags` has values assigned to it later, so const feels wrong, but I guess it works?

Reply:
`issue_tag`'s value holds the reference to an object. Assigning values to the object's properties do not change its value so a `const` was used